include/swift/Basic/BlotMapVector.h

```diff
@@ -139,6 +139,6 @@ class BlotMapVector {
   SmallBlotMapVector() {}
 };
 
-} // end namespace swift
+} // namespace swift
 
-#endif
+#endif /* SWIFT_BASIC_BLOTMAPVECTOR_H */
```

Comment:
Please do not use c style comments in C++ code that is not going to be used in a c context. This is against the LLVM style guide:

http://llvm.org/docs/CodingStandards.html#comment-formatting